e2e-tests/portal/cypress/e2e/pages/utilisation-report-service/previousReports.js

```diff
@@ -0,0 +1,11 @@
+const page = {
+  mainHeading: () => cy.get('[data-cy="main-heading"]'),
+  bodyText: () => cy.get('[data-cy="paragraph"]'),
+  sideNavigation2022Item: () => cy.get('[data-cy="side-navigation-2022"]'),
```

Comment:
Can we refactor the sideNavigation and listItem functions to take in a parameter of year/month which targets the right year/month in the cy.get please